LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/DownlinkMessage.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaTools.LoRaPhysical
+{
+    using System;
+    using LoRaWan;
+
+    /// <summary>
+    /// Model class for a Downlink message for the Basic Station.
+    /// </summary>
+    public class DownlinkMessage
+    {
+        public string DevEui { get; }
+
+        /// This rx delay control the time we need to wait before sending.
```

Comment:
What's this delay expressed in? Can we not use `TimeSpan` so no one has to guess?

Reply:
it's either 1 or 2 (seconds). I can use a timespan if you prefer.

Reply:
As discussed I am not sure timespan is appropriate as only 4 values are possible, let's discuss

Reply:
We could actually use a time 0 -15 would be possible values here.

Reply:
If ok with you I will do it as separate element, I added a comment to specify more the possible values in the meanwhile

Reply:
I think you are already looking at it on another branch